src/main/java/com/fundation/logic/model/SearchVideo.java

```diff
@@ -1,13 +1,14 @@
- /**
-  * Copyright (c) 2019 Jalasoft.
-  * This software is the confidential and proprietary information of Jalasoft.
-  * ("Confidential Information"). You shall not
-  * disclose such Confidential Information and shall use it only in
-  * accordance with the terms of the license agreement you entered into
-  * with Jalasoft.
-  */
+/**
+*
```

Comment:
Please fix the copyright tag according on given example

Reply:
* that was remove  them that was fixed.